src/runtime/analytics-service.js

```diff
@@ -195,7 +195,7 @@ export class AnalyticsService {
    * @param {!../proto/api_messages.AnalyticsEvent} event
    * @return {!AnalyticsRequest}
    */
-  createLogRequest_(event) {
+  createLogRequest(event) {
```

Comment:
I'd keep this private and expose another method
/**
 * @return {!Promise<!AnalyticsContext>}
 */
getContext() {
  return this.start_().then(() => {
    return this.context_;
  }
}

Reply:
Done.